airbyte-integrations/connectors/source-salesloft/source_salesloft/schemas/emails.json

```diff
@@ -0,0 +1,189 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": ["null", "integer"]
```

Comment:
```suggestion
            "type":  "integer"
```
Please double-check which fields are nullable, here for example `id` is probably not nullable. The `updated_at` field should also not be nullable if you want to use it as the cursor field. If Salesloft does not provide this information please at lease set the following field as not nullable:
`id`, `created_at`, `updated_at`

Reply:
Yes please @yannibenoit

Reply:
but @alafanechere we have a similar problem on other schemas 🤔 
id, created_at, updated_at are nullable on the following schemas:
- cadences.json
- people.json
- cadence_memberships.json

Reply:
Made the changes for all schema , let's try out

Reply:
Could you please only change your stream's schema to make this PR atomic, and maybe open a second one for other streams?

Reply:
As this PR is aimed at adding a new stream I find it better to not refactor the other streams in the same PR.
If you're working on PR to change the existing stream feel free to change their schema in this new PR too.
Or if you prefer to align all the schemas in a single PR feel free to do it too.